### DISCUSSION TITLE
comparison with/integration with GluonTS

### DISCUSSION THREAD

**Discussion by timoschowski** [ourownstory/neural_prophet#202] [Feature Requests & Feedback]
This is an awesome project! In GluonTS, we've got some hooks to compare easily with Prophet and so I was wondering if you'd be interesting in this: 
https://github.com/awslabs/gluon-ts/discussions/1213

**Comment by ourownstory** [comment]
Hi @timoschowski, 
We actually actually just got started on a large-scale benchmarking project, including but not limited to GluonTS models. 
Would you like to join the fun? 
Feel free to email me directly at my lastname@stanford.edu address.
